software/script/chameleon_enum.py

```diff
@@ -116,6 +117,8 @@ class Status(enum.IntEnum):
     LF_TAG_OK = 0x40
     # Unable to search for a valid EM410X label
     EM410X_TAG_NO_FOUND = 0x41
+    # Unable to search for a valid LF tag
+    LF_TAG_NO_FOUND = 0x42
```

Comment:
maybe rename `EM410X_TAG_NO_FOUND` to `LF_TAG_NO_FOUND`?

Reply:
> maybe rename `EM410X_TAG_NO_FOUND` to `LF_TAG_NO_FOUND` and reuse `0x41`?

At present, this PR is in the testing phase, and all code formats and naming conventions will be corrected before merging.